Sources/ApolloCodegenLib/Frontend/JavaScript/src/compiler/index.ts

```diff
@@ -413,9 +413,9 @@ export function compileToIR(
     directiveNode: DirectiveNode,
     directiveDef: GraphQLDirective
   ): ir.InclusionCondition | undefined {
-    if (directiveDef == GraphQLIncludeDirective || directiveDef == GraphQLSkipDirective) {      
+    if (directiveDef.name == "include" || directiveDef.name == "skip") {      
```

Comment:
Is there a benefit to this change?

Reply:
It's to get around a bug in graphql-js. `extendSchema` copies the directives, so it's no longer got the same identity as the original `GraphQLIncludeDirective`.